makejinja/app.py

```diff
@@ -22,11 +22,13 @@
 __all__ = ["makejinja"]
 
 
-def makedir(input: Path, output: Path) -> None:
+def makedir(config: Config, input: Path, output: Path) -> None:
     """Create a directory and copy its permissions and times from the input"""
-    if not output.exists():
-        print(f"Create '{output}' directory using '{input}' attributes")
-        output.mkdir()
+
+    print(f"Create '{output}' with metadata from '{input}'")
+    output.mkdir()
+
+    if config.copy_metadata:
```

Comment:
We want to do this after we have copied files into the directory, otherwise the `mtime` gets updated! That's why I had that separate for loop iterating over the `rendered_folders`, sorry for the miscommunication/code filled with trash in my most recent commit.

Reply:
Ok, makes sense. Thanks for the clarification, I will update the code accordingly!